src/backend/Fusion.Resources.Authorization/Requirements/ResourceOwnerRequirement.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Fusion.Authorization;
+using Microsoft.AspNetCore.Authorization;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Authorization.Requirements
+{
+    /// <summary>
+    /// Adjustment of the same type provided by the intergration lib. 
+    /// This will work against a claim added by the local transformer. This will resolve the role provided by the line org for manager responsebility based on SAP data. 
+    /// This update is harder to update in the integration lib claims transformer, due to optimalization.
+    /// </summary>
+    public class BeResourceOwnerRequirement : FusionAuthorizationRequirement, IAuthorizationHandler
+    {
+        public BeResourceOwnerRequirement(string departmentPath, bool includeParents = false, bool includeDescendants = false)
+        {
+            DepartmentPath = departmentPath;
+            IncludeParents = includeParents;
+            IncludeDescendants = includeDescendants;
+        }
+
+        public BeResourceOwnerRequirement()
+        {
+        }
+
+
+        public override string Description => ToString();
+
+        public override string Code => "ResourceOwner";
+
+        public string? DepartmentPath { get; }
+        public bool IncludeParents { get; }
+        public bool IncludeDescendants { get; }
+
+        public Task HandleAsync(AuthorizationHandlerContext context)
+        {
+            var departments = context.User.FindAll(ResourcesClaimTypes.ResourceOwnerForDepartment)
```

Comment:
Copied handler residing in integration lib, so we can use our own claim. 

The original handler uses `var departments = context.User.GetResponsibleForDepartments();` which looks for a claim that is added by the claimstransformer by checking `isResourceOwner` and `user.FullDepartment` which now will be incorrect (one level up).

This will in combination with the local claimstransformer which takes the `Fusion.LineOrg.Manager` role and convert to a the new claim.